nautobot/ipam/models.py

```diff
@@ -652,7 +715,13 @@ class IPAddress(PrimaryModel, StatusModel):
     which has a NAT outside IP, that Interface's Device can use either the inside or outside IP as its primary IP.
     """
 
-    address = IPAddressField(help_text="IPv4 or IPv6 address (with mask)")
+    host = VarbinaryIPField(
+        null=False,
+        db_index=True,
+        help_text="IPv4 or IPv6 host address",
+    )
+    broadcast = VarbinaryIPField(null=False, db_index=True, help_text="IPv4 or IPv6 broadcast address")
```

Comment:
Random, likely incorrect, thought - does an `IPAddress` actually use or need a `broadcast` field? I can see where it makes sense for `Aggregate` and `Prefix`...

Reply:
I waffled on this as well, but the previous implementation uses `inet` which allows you to store the netmask along with the address, so I kept that functionality here